recipes-connectivity/coredns/files/coredns-rules.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+usage (){
+    echo "Manage iptable rule for ARGUS with CoreDNS on any port"
+    echo "  $0 add/del <PodCIDR> <DNSPORT>"
+    echo "Example"
+    echo "  $0 add 10.240.0.0/24"
+    exit
+}
+
+if ! [ "$(id -u)" -eq 0 ]; then
+    echo "Must be run as root"
+    exit
+fi
+
+if ! [ $# -eq 3 ]; then
+    usage
+fi
+
+getGatewayAddress() {
```

Comment:
Replace the method `getGatewayAddress` with the following to handle cases where the CIDR contains 2 or more digits as the last value, e.g. `10.240.0.120/24`, the current function replaces only the last value with `1` and will result in `10.240.0.121`.

```
getGatewayAddress() {
    gat="`echo $1 | cut -d '.' -f1`"
    for n in 2 3; do
        gat="$gat.`echo $1 | cut -d '.' -f$n`"
    done
    gat="$gat.1"

    echo $gat
}
```